engine/mail.asm

```diff
@@ -397,11 +397,11 @@ MailboxPC: ; 0x44806
 
 	ld a, [wCurMessageIndex]
 	ld [wMenuCursorBuffer], a
-	ld a, [OBPals + 8 * 6]
+	ld a, [OBPals palette 6]
 	ld [wMenuScrollPosition], a
 	call ScrollingMenu
 	ld a, [wMenuScrollPosition]
-	ld [OBPals + 8 * 6], a
+	ld [OBPals palette 6], a
```

Comment:
these seem wrong

Reply:
Can you please elaborate? `make compare` is OK, and that's been a common pattern for loading Nth palettes.

Reply:
Why would the menu scroll position be the first byte of the palette?